src/regex_set.rs

```diff
@@ -29,12 +32,23 @@ impl RegexSet {
         &self.items[..]
     }
 
+    pub fn get_unmatched_items(&self) -> Vec<String> {
```

Comment:
nit: I'd call this just `unmatched_items`, and travis is complaining because this needs docs :)